packages/edit-site/src/components/sidebar/navigation-menu-sidebar/navigation-inspector.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState, useEffect, useCallback } from '@wordpress/element';
+import { SelectControl } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+import {
+	__experimentalListView as ListView,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+
+const EMPTY_BLOCKS = [];
+const NAVIGATION_MENUS_QUERY = [ { per_page: -1, status: 'publish' } ];
```

Comment:
As a reminder note, we should probably think though paging and what it should look like. -1 will attempt to fetch all resources, which can be a problem when we have many menus. (page size is limited to 100).